docs/projects/create-project.md

```diff
@@ -12,6 +12,12 @@ A project is a container for all the assets, configuration, and code of a partic
 - [Get a project from DB or create it](#get-or-create)
 - [Deleting a project](#deleting-a-project)
 
+<a id="best-prac-projs"></a>
+## Best Practices
+
+Ensure that no project name is used as the prefix of another project name, since this would affect retrieving pipelines from the KFP.
```

Comment:
```suggestion
Ensure that no project name is used as the prefix of another project name, since this would affect retrieving pipelines from Kubeflow pipelines.
```